posthog/models/feedback/survey.py

```diff
@@ -0,0 +1,63 @@
+from django.db import models
+from posthog.models.utils import UUIDModel
+
+
+class Survey(UUIDModel):
+    class SurveyType(models.TextChoices):
+        POPOVER = "popover", "popover"
+        BUTTON = "button", "button"
+        EMAIL = "email", "email"
+        FULL_SCREEN = "full_screen", "full screen"
+
+    class Meta:
+        constraints = [models.UniqueConstraint(fields=["team", "name"], name="unique survey name for team")]
+
+    team: models.ForeignKey = models.ForeignKey(
+        "posthog.Team", on_delete=models.CASCADE, related_name="surveys", related_query_name="survey"
+    )
+    name: models.CharField = models.CharField(max_length=400)
+    description: models.TextField = models.TextField(blank=True)
+    linked_flag: models.ForeignKey = models.ForeignKey(
+        "posthog.FeatureFlag",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="surveys_linked_flag",
+        related_query_name="survey",
+    )
+    targeting_flag: models.ForeignKey = models.ForeignKey(
+        "posthog.FeatureFlag",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="surveys_targeting_flag",
+        related_query_name="survey",
+    )
+    type: models.CharField = models.CharField(max_length=40, choices=SurveyType.choices)
+
+    # { url: 'posthog.com/feature', selector: null, triggers: [{}] #similar to cohort behavioral filters for now?}
+    conditions: models.JSONField = models.JSONField(blank=True, null=True)
+
+    # class SurveyQuestionType(models.TextChoices):
```

Comment:
Is this meant to be

Reply:
if I remove just the comments later that won't affect any migrations right?

Reply:
it would, we'd just move it out of the model since questions are a json field anyway.